gslib/utils/wrapped_credentials.py

```diff
@@ -125,8 +134,13 @@ def from_json(cls, json_data):
     # Rebuild the credentials.
     base = data.get("_base")
     # Init base cred.
-    external_account_creds = _get_external_account_credentials_from_info(base)
-    creds = cls(external_account_creds)
+    base_creds = None
+    if base.get('type') == 'external_account':
+      base_creds = _get_external_account_credentials_from_info(base)
+    elif base.get('type') == 'external_account_authorized_user':
```

Comment:
Do you have an enum or constants list somewhere in the auth package you can use for these strings?

Reply:
There isn't, but that is a good idea to include.  I'll talk to @leosiracusa